dockers/base-cuda/Dockerfile

```diff
@@ -21,6 +21,7 @@ FROM nvidia/cuda:${CUDA_VERSION}-devel-ubuntu${UBUNTU_VERSION}
 ARG PYTHON_VERSION=3.9
 ARG PYTORCH_VERSION=1.13
 ARG MAX_ALLOWED_NCCL=2.16.2
+ARG PRE_RELEASE
```

Comment:
I would not include pre option to dockers, lets keep it simple :)

Reply:
This is still left from when I wanted to follow another path for GPU CI.